pkg/nsx/util/utils.go

```diff
@@ -271,3 +271,12 @@ func MergeAddressByPort(portAddressOriginal []PortAddress) []PortAddress {
 	}
 	return portAddress
 }
+
+func ParseVPCPath(nsxResourcePath string) (orgID string, projectID string, vpcID string, resourceID string) {
+	paras := strings.Split(nsxResourcePath, "/")
+	orgID = paras[2]
+	projectID = paras[4]
+	vpcID = paras[6]
+	resourceID = paras[8]
```

Comment:
Can we assert that resourceID is always paras[8]? Taking subnet port as example, paras[8] is not port_id, for most cases resourceID should be the last element of paras?

`/orgs/{{org_id}}/projects/{{pro_id}}/vpcs/{{vpc_id}}/subnets/{{subnet_id}}/ports/{{port_id}}`

Reply:
I abstracted the logic into https://github.com/vmware-tanzu/nsx-operator/pull/237, will rebase here after it is merged.